File: RepoTransBench/target_projects/Java/Go/caoym_jjvm/public_tests/jvmfieldmethod_public_test.go
package public_tests

import (
	"testing"
)

type JvmField struct {
	name  string
	typ   string
	value interface{}
	_stat bool
}

func NewJvmField(name, typ string, value interface{}, stat bool) *JvmField {
	return &JvmField{name: name, typ: typ, value: value, _stat: stat}
}

func (f *JvmField) getValue() interface{}            { return f.value }
func (f *JvmField) setValue(val interface{})         { f.value = val }

func TestAccessDifferentPrimitiveFields(t *testing.T) {
	intField := NewJvmField("score", "I", 7, false)
	boolField := NewJvmField("visible", "Z", true, false)

	if intField.getValue() != 7 {
		t.Errorf("Expected 7, got %v", intField.getValue())
	}
	if boolField.getValue() != true {
		t.Errorf("Expected true, got %v", boolField.getValue())
	}

	intField.setValue(42)
	if intField.getValue() != 42 {
		t.Errorf("Expected 42 after set, got %v", intField.getValue())
	}

	boolField.setValue(false)
	if boolField.getValue() != false {
		t.Errorf("Expected false after set, got %v", boolField.getValue())
	}
}

func TestStringFieldDifferentValue(t *testing.T) {
	strField := NewJvmField("owner", "Ljava/lang/String;", "robot", false)
	if strField.getValue() != "robot" {
		t.Errorf("Expected 'robot', got %v", strField.getValue())
	}

	strField.setValue("android")
	if strField.getValue() != "android" {
		t.Errorf("Expected 'android' after set, got %v", strField.getValue())
	}
}

File: RepoTransBench/target_projects/Java/C#/caoym_jjvm/public_tests/JvmFieldMethodPublicTest.cs
using System;
using Xunit;
using CaoymJjvm.Lang;

namespace CaoymJjvm.PublicTests
{
    public class JvmFieldMethodPublicTest
    {
        [Fact]
        public void TestAccessDifferentPrimitiveFields()
        {
            var intField = new JvmField("score", "I", 7, false);
            var boolField = new JvmField("visible", "Z", true, false);

            Assert.Equal(7, intField.Value);
            Assert.True((bool)boolField.Value);

            intField.Value = 42;
            Assert.Equal(42, intField.Value);

            boolField.Value = false;
            Assert.False((bool)boolField.Value);
        }

        [Fact]
        public void TestStringFieldDifferentValue()
        {
            var strField = new JvmField("owner", "Ljava/lang/String;", "robot", false);
            Assert.Equal("robot", strField.Value);

            strField.Value = "android";
            Assert.Equal("android", strField.Value);
        }
    }
}